.circleci/config.yml

```diff
@@ -211,17 +211,17 @@ jobs:
           at: /app
       - restore_cache:
           keys:
-            - npm-dependencies-{{ checksum "package-lock.json" }}
+            - yarn-dependencies-{{ checksum "yarn.lock" }}
       - run:
-          name: Install npm dependencies
-          command: npm install --no-save
+          name: Install yarn dependencies
+          command: yarn
```

Comment:
Didn't know `yarn` run `install` when argument-less! :+1:

Reply:
It's the only thing I know about yarn sadly 🤣